setup.py

```diff
@@ -1,3 +1,20 @@
+# (C) Copyright ${year} core.ai (https://core.ai/) 
```

Comment:
There are 4 duplicate commits in the PR.  It is a good practice to squash redundant commits.

https://blog.carbonfive.com/always-squash-and-rebase-your-git-commits/

Reply:
Done